include/glow/Support/Error.h

```diff
@@ -462,6 +462,7 @@ class GlowExpected final
   template <typename TT>
   friend TT detail::exitOnError(const char *fileName, size_t lineNumber,
```

Comment:
Once you have touched Error.h file, can you fix the Macros problem? Macros must be either outside glow namespace or any reference inside macro must have the fully decorated name "glow::", check marco: RETURN_IF_ERR and FAIL_TEST_IF_ERR

Reply:
@putivsky for the macros it looks like this is the case, can you point me to where it's not and I'll fix it.
GlowExpected is within the glow namespace so this is ok.

Reply:
Image you are calling Macro outside glow namespace. It will fail and you cannot call glow::Marco, the only way to use it it define "using namespace glow" - very inconvenient. Even Marco uses glow functions inside glow namespace it must call it will fully decorated name, i.e. glow::Error.
I have found two macros: RETURN_IF_ERR and FAIL_TEST_IF_ERR

In general it's a good practice to declare macros outside any namespaces.